packages/realm-react/src/RealmProvider.tsx

```diff
@@ -158,14 +162,18 @@ export function createRealmProviderFromConfig(
       }
     }, [realm]);
 
+    const [progress, setProgress] = useState<number | null>(null);
```

Comment:
As per my previous comment.

```suggestion
    const [progress, setProgress] = useState<number>(0);
```

Reply:
thing is depending on the type of progress the next value could either be 1 or 0, kind of hard to come up with a good default to show right away